### PR TITLE
aghs and shard buff

### PR DESCRIPTION
**Pull request by Argn0**
https://deploy-preview-2716--opendota-staging.netlify.app/matches/5748344216

### CODE REVIEW

**Comment by howardchung** [comment]
Looks nice, although without tooltips it's sort of unclear what was upgraded

**Comment by Argn0** [comment]
Hmm, I think these tooltips hurt performance.

**Comment by Argn0** [comment]
Hmm, I guess it's alright. Do you notice longer loading times?

**Comment by howardchung** [comment]
Perf seems fine to me

**Comment by atilla4455** [comment]
Actually I should put the Tooltip constants outside the displayFn

File: src/components/Match/matchColumns.jsx [hunk 1]
```diff
@@ -374,14 +375,48 @@ export default (strings) => {
...
       .concat(match.players.map(player => player.permanent_buffs && player.permanent_buffs.length).reduce(sum, 0) > 0
         ? {
           displayName: strings.th_permanent_buffs,
           tooltip: strings.tooltip_permanent_buffs,
           field: 'permanent_buffs',
           width: 135,
           displayFn: row =>
-            (row.permanent_buffs && row.permanent_buffs.length > 0 ? row.permanent_buffs.map(buff => inflictorWithValue(buffs[buff.permanent_buff], buff.stack_count, 'buff')) : '-'),
+            (row.permanent_buffs && row.permanent_buffs.length > 0 ? row.permanent_buffs
+            .filter(b => b.permanent_buff !== 2 && b.permanent_buff !== 12).map(buff => inflictorWithValue(buffs[buff.permanent_buff], buff.stack_count, 'buff')) : '-'),
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx:419]: probably want to put 2 and 12 into named constants for readability

Comment (Argn0) [reply src/components/Match/matchColumns.jsx]: done

**Comment by Argn0** [comment]
changed abbreviation "agh" to "scepter"

File: src/components/Match/matchColumns.jsx [hunk 2]
```diff
@@ -374,14 +380,43 @@ export default (strings) => {
...
+          displayFn: row => 
+            <StyledAghanimsBuffs>
+              <ReactTooltip id="scepter" effect="solid" place="left">
+                {scepterTooltip}
+              </ReactTooltip>
+              <ReactTooltip id="shard" effect="solid" place="left">
+                {shardTooltip}
+              </ReactTooltip>
+              <img
+                src={`/assets/images/dota2/scepter_${row.permanent_buffs && row.permanent_buffs.some(b => b.permanent_buff === 2) ? '1' : '0'}.png`}
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx:397]: you can also use the constants here

**Review by howardchung — APPROVED** [review]

**Pull request merged**